akka-typed/src/main/scala/akka/typed/package.scala

```diff
@@ -1,6 +1,8 @@
-package akka.typed
+package akka
 
-package object internal {
+import akka.typed.internal.ActorRefImpl
+
+package object typed {
```

Comment:
hmm, these should probably be `private[typed]`—the members are not accessible anyway, but better not confuse people with nonsensical extension methods